### ISSUE TITLE
Discussion - Not an issue

### ISSUE THREAD

**Issue by ghost** [akalenuk/the_namingless_programming_language#1]
Thanks for sharing this.

**Comment by akalenuk** [comment]
Thank you!

**Comment by ghost** [comment]
I had written a write up, thought it was a junk and deleted it. Your project and idea is neat, very inspiring and sparked me up.

Anyways, let me share my mind voice on this technique, and an imaginary technique I have.

Challenges with programming language - 

1. Its a distraction to solve a problem.
2. mind has to understand a language, to communicate the ideas back.
3. If a language is created by a math, and expect an artist to use his ideas, the result will not match original idea.
4. What if I create my own programming language. and it is a circle of repeated cycle.

Here is a vague thought I had. Programming language is made of tree to represent the language, but the instruction execution is a graph, I mean, function calls are jumpy, events callbacks trigger certain **place** in code at particular **point** of **time**.

So instead of trying to create a new programming language based on tree, how about build something based on space and time unit, because time, space is the one of few things common between a computer and human. So I tried to define from scratch.

1. time runs 
2. space stays
3. display is show or hide at different point of space at a time
4. storage is read or write at different point of space at a time

Any creation out of computer must have to fall on these units. 

In other words, creating a nameless programming language as you did, and my thoughts above, does it sound like it is aligning ? because that's why your project gained my attention, because what I believe is nouns, are useless, in your project it is name. In truth, nouns are verb from past. In case of computer programming language, these nouns are and have become useless because of the disconnect we have with fundamental units of space, time, storage, display. 

p.s - I got deviated, I was trying to say or ask, what could be the next clean up. From named programming language, to nameless programming language. Next is what?
